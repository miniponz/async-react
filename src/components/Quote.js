import React from 'react';
import PropTypes from 'prop-types';

function Quote({ character, quote, image }) {
  return (
    <section>
      <h3>{character}</h3>
      <img src={image}></img>
      <p>{quote}</p>
    </section>
  );
}

Quote.propTypes = {
  character: PropTypes.string.isRequired,
  quote: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Quote;

