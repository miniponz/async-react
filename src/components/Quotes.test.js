import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('quote component', () => {
  it('renders quotes', () => {
    const wrapper = shallow(<Quote character={''} quote={''} image={''}/>);
    expect(wrapper).toMatchSnapshot();
  });   
});
