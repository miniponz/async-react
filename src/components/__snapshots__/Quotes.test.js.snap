// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quote component renders quotes 1`] = `
<section>
  <h3 />
  <img
    src=""
  />
  <p />
</section>
`;
